examples/pytorch/AxHyperOptimizationPTL/conda.yaml

```diff
@@ -6,15 +6,15 @@ dependencies:
   - pip:
       - mlflow
       - pytorch-lightning==1.7.6
+      # typeguard is used for type validation in the ax-platform code base. 3.0.0 release has
+      # breaking changes that need to be resolved in ax. Remove this pin when
+      # https://github.com/facebook/Ax/issues/1509 is addressed
+      - typeguard<3.0.0
       - ax-platform
       - torchvision>=0.9.1
       - torch>=1.9.0
-      - protobuf<4.0.0
       # gyptorch 1.9.x is incompatible with the versions of botorch
       # required by many versions of pytorch
       - gpytorch<1.9.0
-      # Pinning pandas version less than 1.4.4 due to https://github.com/facebook/Ax/issues/1153
-      - pandas<=1.4.4
```

Comment:
Latest versions of ax-platform seem to have these issues resolved, according to the linked issue comments and release notes.

Reply:
... and they didn't address them. Their latest release still has the issue with columns unable to be referenced as a set. Putting this pinning back in.